examples/Model.elm

```diff
@@ -44,14 +44,14 @@ type alias ClientFormData =
 
 
 type alias Model =
-    { error : Maybe String
-    , client : Maybe Kinto.Client
-    , pager : Maybe (Kinto.Pager Record)
+    { maybeError : Maybe String
+    , maybeClient : Maybe Kinto.Client
+    , maybePager : Maybe (Kinto.Pager Record)
     , formData : FormData
     , clientFormData : ClientFormData
     , currentTime : Posix
     , sort : Sort
-    , limit : Maybe Int
+    , maybeLimit : Maybe Int
```

Comment:
This seems quite strange to me to be honest.

Reply:
how so? Isn't it more explicit? And it flows naturally when using the `case` and means we don't need to use single letter names.